scripts/run_ansible_test

```diff
@@ -52,6 +52,6 @@ ${ansible_test} sanity --test validate-modules
 if [ -d tests/integration ]; then
     ${ansible_test} integration --color=yes
 fi
-if [ -d tests/units ]; then
-    ${ansible_test} units --color=yes
+if [ -d tests/unit ]; then
```

Comment:
actually, according to the doc, it should indeed be tests/units:
https://docs.ansible.com/ansible/latest/dev_guide/testing_units.html

also shown in the ansible repository:
https://github.com/ansible/ansible/tree/devel/test/units

fun, because it used to run anyway...

Reply:
@cjeanner are you sure? https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_internal/provider/layout/collection.py#L123

This is the ansible-test output when I use tests/units instead of tests/unit:
`❯ ansible-test units
WARNING: Rename "tests/units/" to "tests/unit/" to run unit tests.
WARNING: All targets skipped.
`

Reply:
consistency.. :rofl: